<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>路径查询2.3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="3,1">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="0" column="0">
     <widget class="Map" name="map"/>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>消息栏</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="NewText">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="tabStopWidth">
            <number>80</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="mapChoice">
         <item>
          <property name="text">
           <string>斜线图</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>直线图</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="choices">
         <item>
          <property name="text">
           <string>最便宜的路径</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>最省时间的路径（不考虑等车）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>最省时间的路径（考虑等车）</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QVBoxLayout" name="_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>站点A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="StationA"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>-&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>站点B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="StationB"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="computer">
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="loadin">
           <property name="text">
            <string>导入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loudout">
           <property name="text">
            <string>导出</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Map</class>
   <extends>QGraphicsView</extends>
   <header location="global">map.h</header>
   <slots>
    <slot>paint()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <signal>change()</signal>
 </slots>
</ui>
